remotes/mount.go

```diff
@@ -151,16 +151,25 @@ func (r *remoteReaderAt) ReadAt(p []byte, off int64) (int, error) {
 }
 
 type descriptorAccumulator struct {
-	descriptors []ocischemav1.Descriptor
+	descriptors     []ocischemav1.Descriptor
+	childrenHandler images.HandlerFunc
+	resolver        docker.ResolverBlobMounter
+	targetRepo      string
```

Comment:
You _could_ avoid overloading this `descriptorAccumulator` by instead having a new/separate `images.Handler` implementation which returns `ErrSkipDesc` and have that at the head of your `images.Handlers()` chain.

Reply:
👍

Reply:
Invalidating by upcoming PR for events :)